src/stores/mod.rs

```diff
@@ -0,0 +1,233 @@
+use std::cmp::Eq;
+use std::collections::hash_map::Entry;
+use std::collections::HashMap;
+use std::hash::Hash;
+
+use super::Cached;
+
+#[cfg(feature = "async")]
+use {super::CachedAsync, async_trait::async_trait, futures::Future};
+
+pub mod sized;
+pub mod timed;
+pub mod timed_sized;
+pub mod unbound;
+
+pub type UnboundCache<K, V> = unbound::UnboundCache<K, V>;
+pub type SizedCache<K, V> = sized::SizedCache<K, V>;
+pub type TimedSizedCache<K, V> = timed_sized::TimedSizedCache<K, V>;
+pub type TimedCache<K, V> = timed::TimedCache<K, V>;
+
+/// Limited functionality doubly linked list using Vec as storage.
+#[derive(Clone, Debug)]
+struct LRUList<T> {
```

Comment:
could we move this to a `pub(crate) mod lru_list` to clear out this file while we're at it?

Reply:
My plan was to keep everything inside the mod.rs file, except the public store types.
But that's fine, so I will put it next to the `lib.rs`.

Any preference for `impl<K: Hash + Eq, V> Cached<K, V> for HashMap<K, V>`?